core/src/manifest.rs

```diff
@@ -160,26 +160,34 @@ impl ManifestValidationResult {
 
 /// Manifest containing [`ManifestRequest`] and [`ManifestResponse`]
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq, From)]
-// #[serde(rename_all = "camelCase")]
+#[serde(rename_all = "camelCase")]
 pub struct Manifest {
-  // /// Manifest version
-  // pub manifest_version: String,
-  // /// ID of the manifest
-  // pub id:               String,
-  // /// Title of the manifest
-  // pub title:            String,
-  // /// Description of the manifest
-  // pub description:      String,
+  /// Manifest version
+  pub manifest_version: String,
```

Comment:
is it worth having this be "typed" in any way? 

probably not, i'm sure a `String` is just fine.